app/models/event.rb

```diff
@@ -117,18 +119,18 @@ class Event < ActiveRecord::Base
       type: { type: :keyword },
       id: { type: :keyword },
       uid: { type: :keyword },
-      proxyIdentifiers: { type: :keyword },
-      datePublished: { type: :date, format: "date_optional_time||yyyy-MM-dd||yyyy-MM||yyyy", ignore_malformed: true },
```

Comment:
@mfenner these are the attributes we cannot access because of the casing. 

If we do not change the mapping, we could:

A) modify all the values in the database to camelCase (either re importing all events or transforming all of them)
B) change the indexJob so the transformation happens there during every index.

Reply:
I think we need both.